src/khoj/routers/storage.py

```diff
@@ -33,3 +33,25 @@ def upload_image(image: bytes, user_id: uuid.UUID):
     except Exception as e:
         logger.error(f"Failed to upload image to S3: {e}")
         return None
+
+
+AWS_USER_UPLOADED_IMAGES = os.getenv("AWS_USER_UPLOADED_IMAGES")
+
+
+def upload_image_bucket(image: bytes, user_id: uuid.UUID):
+    """Upload the image to the S3 bucket"""
+    if not aws_enabled:
+        logger.info("AWS is not enabled. Skipping image upload")
+        return None
+
+    image_key = f"{user_id}/{uuid.uuid4()}.webp"
+    try:
+        s3_client.put_object(
+            Bucket=AWS_USER_UPLOADED_IMAGES, Key=image_key, Body=image, ACL="public-read", ContentType="image/webp"
+        )
+        # open using default browser
+        # os.startfile(f"https://{bucket_name}.s3.amazonaws.com/{image_key}")
+        return f"https://{AWS_USER_UPLOADED_IMAGES}.s3.amazonaws.com/{image_key}"
```

Comment:
This is ok for testing, but we should make this
```suggestion
        return f"https://{AWS_USER_UPLOADED_IMAGES}/{image_key}"
```